common/parallel/FileSystemCrawler.go

```diff
@@ -125,7 +131,9 @@ func enumerateOneFileSystemDirectory(dir Directory, enqueueDir func(Directory),
 
 	d, err := os.Open(dirString) // for directories, we don't need a special open with FILE_FLAG_BACKUP_SEMANTICS, because directory opening uses FindFirst which doesn't need that flag. https://blog.differentpla.net/blog/2007/05/25/findfirstfile-and-se_backup_name
 	if err != nil {
-		return err
+		// FileInfo value being nil should mean that the FileSystemEntry refers to a directory.
+		enqueueOutput(FileSystemEntry{dirString, nil}, err)
+		return nil
```

Comment:
nit, add a comment explaining why we return nil

// Since we have already enqueued the failed enumeration entry, return nil error to avoid duplicate queueing by workerLoop().

also at L151